### PR TITLE
Typo fix (plural to singular) on msg ext search commands

### PR DESCRIPTION
**Pull request by andrewconnell**
Pretty sure this is supposed to be singular, not plural

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md](https://github.com/MicrosoftDocs/msteams-docs/blob/720cd092d7e5a7fb74a496d6e1fb9e14bcd0e685/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Fsearch-commands%2Frespond-to-search.md) | 91 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9d7467cc-a94b-419d-ba7b-7e6a098ba8d6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ ms.localizationpriority: medium
 
 [!include[v4-to-v3-SDK-pointer](~/includes/v4-to-v3-pointer-me.md)]
 
-After the user submits the search command, your web service receives a `composeExtensions/query` invoke message that contains a `value` object with the search parameters. The invoke is triggered with the following conditions:
+After the user submits the search command, your web service receives a `composeExtension/query` invoke message that contains a `value` object with the search parameters. The invoke is triggered with the following conditions:
```
Comment (v-deabira) [inline msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md:13]: Hi @andrewconnell, 

The name of the property in manifest schema is "composeExtensions".
Sharing the link for your reference:
https://learn.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema#composeextensions
image

Comment (andrewconnell) [reply msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md:13]: Correct, but the change in the PR is not referencing the app manifest. The article is talking about the type of the invoke message the bot framework receives & sends to the web service. That's `composeExtension/query`. 

Look at the same doc I submitted the PR for. The PR fixes the typo in the first paragraph, but if you scroll down to the first JSON code listing, you'll see the correct usage:

```json
{
  "type": "invoke",
  "name": "composeExtension/query",
  ..
}
```

Comment (v-deabira) [reply msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md:13]: Okay. Thanks for the clarity.

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4945f07f5a4290591fb4542f2360a4098fadeede/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Fsearch-commands%2Frespond-to-search.md) | 91 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/99abb08c-9340-4855-9b37-570133d1ad99) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**